src/containers/signupForm/index.tsx

```diff
@@ -0,0 +1,509 @@
+              style={{ display: 'inline-block', width: 'calc(50% - 8px)' }}
+            >
+              <DatePicker />
+            </Form.Item>
+
+            <Form.Item
+              name="address"
+              label="Address"
+              className="stacked-inputs"
+              rules={[{ required: true, message: 'Please input your address' }]}
+            >
+              <Input placeholder="Address" />
+            </Form.Item>
+            <Form.Item
+              name="city"
+              className="inline-block-third"
+              rules={[{ required: true, message: 'Please input city' }]}
+            >
+              <Input placeholder="City" />
+            </Form.Item>
+            <Form.Item
+              name="state"
+              className="inline-block-third"
+              rules={[
+                {
+                  required: true,
+                  pattern: new RegExp(/^[A-Za-z]{2}/),
+                  message: 'Please input state',
+                },
+              ]}
+            >
+              <Input placeholder="State" />
+            </Form.Item>
+            <Form.Item
+              name="zip"
+              className="block-third"
+              rules={[
+                {
+                  required: true,
+                  pattern: new RegExp(/^[^0-9]*(?:(\d)[^0-9]*){5}$/),
+                  message: 'Please input a valid five-digit zip code',
+                },
+              ]}
+            >
+              <Input placeholder="Zip Code" />
+            </Form.Item>
+
+            <Form.Item label="Referrer (if applicable)" name="referrer">
+              <Select
+                showSearch
+                style={{ width: 200 }}
+                placeholder="None"
+                optionFilterProp="children"
+                filterOption={(input, option) =>
+                  option &&
+                  option.children &&
+                  option.children.toLowerCase().indexOf(input.toLowerCase()) >=
+                    0
+                }
+              >
+                <Option value="None">None</Option>
+                <Option value="TWP">TWP</Option>
+                <Option value="NBCR">NBCR</Option>
+                <Option value="LLB">LLB</Option>
+                <Option value="Other">Other</Option>
+              </Select>
+            </Form.Item>
+
+            <Form.Item label="Allergies (if applicable)" name="allergies">
+              <TextArea rows={3} placeholder="Allergies" />
+            </Form.Item>
+
+            <Form.Item label="Diagnosis" name="diagnosis">
+              <TextArea rows={1} placeholder="Diagnosis" />
+            </Form.Item>
+
+            <Form.Item label="Medication (if applicable)" name="medications">
+              <TextArea rows={1} placeholder="Medication" />
+            </Form.Item>
+
+            <Form.Item label="Other Notes" name="otherNotes">
+              <TextArea rows={3} placeholder="Other Notes" />
+            </Form.Item>
```

Comment:
Aren't these on contacts, not signup?

Reply:
Yep but our Signup request takes in information for the main contact anyway! Hey, I didn't design the API 🤷